src/main/java/org/gridsuite/loadflow/server/repositories/ComponentResultRepository.java

```diff
@@ -23,4 +25,9 @@
 @Repository
 public interface ComponentResultRepository extends JpaRepository<ComponentResultEntity, UUID>, JpaSpecificationExecutor<ComponentResultEntity> {
     List<ComponentResultEntity> findAll(Specification specification, Sort sort);
+
+    @Query(value = "SELECT distinct l.status from ComponentResultEntity as l " +
+            "where l.loadFlowResult.resultUuid = :resultUuid " +
+            "order by l.status")
+    List<LoadFlowResult.ComponentResult.Status> findComputingStatus(UUID resultUuid);
```

Comment:
The plural form of "status" is "Statuses". So given that we return a list I think the function should be called `findComputingStatus`.

Reply:
"Statuses" doesn't make sense
 i will be return the List of the possible values declared in the computing status enum and not a list of status ...

Reply:
The return type is literally `List<LoadFlowResult.ComponentResult.Status>`. And it doesn't look like "possible values", it looks like real statuses from a result with a specific resultUuid. Or I really don't understand the function.

By the way a bit later you write this with an 's' : 

`List<LimitViolationType> findLimitType**s**(UUID resultUuid);`

Looks like the same function to me.